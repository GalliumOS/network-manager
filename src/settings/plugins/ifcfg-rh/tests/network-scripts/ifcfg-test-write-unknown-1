FOO='val
  bar=3'
wrong line
  F2=b
  F3='b
XXX=adf'
  XXX2=val2
'
