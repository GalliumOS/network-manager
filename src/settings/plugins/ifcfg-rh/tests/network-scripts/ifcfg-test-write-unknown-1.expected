FOO=
#NM: FOO='val
bar=
#NM:   bar=3'
#NM: wrong line
  F2=b
F3=
#NM:   F3='b
XXX=
#NM: XXX=adf'
  XXX2=val2
#NM: '
